QuantConnect.IBAutomater/IBAutomater.cs

```diff
@@ -101,72 +107,104 @@ public IBAutomater(string ibDirectory, string ibVersion, string userName, string
         /// </summary>
         /// <param name="waitForExit">true if it should wait for the IB Gateway process to exit</param>
         /// <remarks>The IB Gateway application will be launched</remarks>
-        public void Start(bool waitForExit)
+        public bool Start(bool waitForExit)
         {
-            if (IsLinux)
+            if (IsRunning())
             {
-                // debug testing
-                if (!File.Exists("IBAutomater.sh"))
-                {
-                    throw new Exception($"IBAutomater.sh file not found - current directory: {Directory.GetCurrentDirectory()}");
-                }
+                return true;
+            }
+
+            if (_isStarting)
+            {
+                OutputDataReceived?.Invoke(this, new OutputDataReceivedEventArgs("IBAutomater is already starting."));
+                return false;
```

Comment:
Wonder if this should be `true`? The user wants to start the `automater`, if he is already starting, seems like a happy path 🤔